lib/filters/index.js

```diff
@@ -140,11 +144,12 @@ module.exports = ruleSource => {
     };
   });
 
-  return (url, callback) => {
+  return (payload, callback) => {
```

Comment:
isn't it the `url` from above?

Reply:
Confusingly, no. It's a payload object that has a `url` key inside it.

Reply:
It's hard to follow but the `tests` functions take a `req`, which is a socket payload: 
https://github.com/snyk/broker/blob/907a63baa2e183f612ab0824beea39b1876b84d4/lib/relay.js#L99-L105

Would be good to refactor these variable names in a separate PR